registry/storage/driver/s3-aws/s3_test.go

```diff
@@ -30,6 +30,7 @@ func init() {
 	secure := os.Getenv("S3_SECURE")
 	region := os.Getenv("AWS_REGION")
 	root, err := ioutil.TempDir("", "driver-")
+	regionEndpoint := ""
```

Comment:
It should be possible to test the driver against S3 compatible APIs, so how about changing this to `regionEndpoint := os.Getenv("REGION_ENDPOINT")`?